### PR TITLE
Add profile settings feature

### PR DESCRIPTION
**Pull request by nebadesmondc**

### CODE REVIEW

File: android/app/src/main/java/com/queatz/ailaai/ui/dialogs/ProfileSettingsDialog.kt [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+                            stringResource(R.string.profile_setting),
+                            style = MaterialTheme.typography.titleLarge,
+                            modifier = Modifier
+                               .padding(bottom = 1.pad)
+                        )
+                    }
+                    Check(
+                        checked = profileConfig.showGroups ?: true,
+                        onCheckChanged = {
+                            profileConfig.showGroups =  it
```
Comment (Queatz) [inline android/app/src/main/java/com/queatz/ailaai/ui/dialogs/ProfileSettingsDialog.kt]: nitpick: double space after the '=`

File: web/src/jsMain/kotlin/components/ProfilePage.kt [hunk 2]
```diff
@@ -413,7 +413,7 @@ fun ProfilePage(personId: String? = null, url: String? = null, onProfile: (Perso
                                     }
                                 }
 
-                                if (groups.isNotEmpty()) {
+                                if (profile.profile.config?.showGroups == true) {
```
Comment (Queatz) [inline web/src/jsMain/kotlin/components/ProfilePage.kt]: We still want to hide the section when `groups.isNotEmpty()` too, also this should be `showGroups != false` same as in Android

File: web/src/jsMain/kotlin/components/ProfilePage.kt [hunk 3]
```diff
@@ -413,22 +413,24 @@ fun ProfilePage(personId: String? = null, url: String? = null, onProfile: (Perso
...
                                 }
 
-                                if (profile.profile.config?.showGroups == true) {
-                                    Span({
-                                        style {
-                                            marginTop(1.r)
-                                            fontWeight("bold")
-                                            fontSize(24.px)
+                                if (profile.profile.config?.showGroups != false) {
+                                    if (groups.isNotEmpty()) {
```
Comment (Queatz) [inline web/src/jsMain/kotlin/components/ProfilePage.kt]: We can still combine the conditions into 1 if statement 😃

Comment (nebadesmondc) [reply web/src/jsMain/kotlin/components/ProfilePage.kt]: Oh yes!
Let me fix that

**Review by Queatz — APPROVED** [review]
LGTM!

**Pull request merged**